apps/frontend/src/components/ThisWeek/ThisWeekComponent.tsx

```diff
@@ -0,0 +1,120 @@
+"use client"
+
+import React from "react"
+
+import { Icon } from "@iconify-icon/react"
+import { LeftChevron, RightChevron } from "@/src/lib/icons/Navigation"
+
+const ThisWeekComponent: React.FC = () => {
+  return (
+    <div className="flex flex-col gap-8">
+      <div className="flex items-center gap-8 text-xs">
+        <h1 className="text-foreground text-2xl">Week 1</h1>
+        <div className="flex gap-4">
+          <p>0/6 completed</p>
+          <p>0%</p>
+          <p>aug 19th - aug 26th</p>
+        </div>
+        <div className="flex items-center justify-between gap-4">
+          <button onClick={() => console.log("left")} className="p-2">
+            <LeftChevron />
+          </button>
+          <button onClick={() => console.log("right")} className="p-2">
+            <RightChevron />
+          </button>
```

Comment:
A separate component can be create for this which we can render at the client side